Sources/MapboxCoreNavigation/CoreNavigationNavigator.swift

```diff
@@ -54,6 +54,14 @@ class Navigator {
     // Used in tests to recreate the navigator
     static func _recreateNavigator() { _navigator = nil }
     
+    /**
+     Profile setting, used for selecting tiles type for navigation.
+     
+     This property can only be modified before creating `Navigator` shared instance, all
+     further changes to this property will have no effect. Defaults to `automobile`.
+     */
+    static var datasetProfile: ProfileIdentifier = .automobile
```

Comment:
Rename to `datasetProfileIdentifier` for consistency with the actual type, which is `ProfileIdentifier`, not `DatasetProfile`.